scripts/prepare_sector_network.py

```diff
@@ -307,30 +383,65 @@ def add_hydrogen(n, costs):
             h2_pot.index = cavern_nodes.index
             # h2_pot = h2_pot * cavern_nodes.fraction * 1e6
 
+            # n.add("Carrier", "H2 UHS")
+
+            n.madd(
+                "Bus",
+                nodes + " H2 UHS",
+                location=nodes,
+                carrier="H2 UHS",
+                x=n.buses.loc[list(nodes)].x.values,
+                y=n.buses.loc[list(nodes)].y.values,
+            )
+
             n.madd(
                 "Store",
-                cavern_nodes.index + " H2 Store",
-                bus=cavern_nodes.index + " H2",
+                cavern_nodes.index + " H2 UHS",
+                bus=cavern_nodes.index + " H2 UHS",
                 e_nom_extendable=True,
                 e_nom_max=h2_pot.values,
                 e_cyclic=True,
-                carrier="H2 Store",
+                carrier="H2 UHS",
                 capital_cost=h2_capital_cost,
             )
 
+            n.madd(
+                "Link",
+                nodes + " H2 UHS charger",
+                bus0=nodes,
+                bus1=nodes + " H2 UHS",
+                carrier="H2 UHS charger",
+                # efficiency=costs.at["battery inverter", "efficiency"] ** 0.5,
+                capital_cost=0,
+                p_nom_extendable=True,
+                # lifetime=costs.at["battery inverter", "lifetime"],
+            )
+
+            n.madd(
+                "Link",
+                nodes + " H2 UHS discharger",
+                bus0=nodes,
+                bus1=nodes + " H2 UHS",
+                carrier="H2 UHS discharger",
+                efficiency=0.83,

```

Comment:
Make it consistent between consum data and none custom data.

Reply:
Dealt with